.gitignore

```
@@ -12,9 +12,14 @@ __pycache__
 .pytest_cache
 .python-version
 *.egg-info
+.venv/
 dist
+.venv
```

Preserve alphabetical order and remove unnecessary duplicate entry:
```suggestion
.venv
*.egg-info
dist
```

---

resolved with [9d032df](https://github.com/cisagov/windows-commando-packer/pull/10/commits/9d032dfac620ac8c6eb7411233352691b09ad6e7)

---

@dav3r just following up on this. anything else i should do? thanks

---

https://github.com/cisagov/windows-commando-packer/pull/10/commits/9d032dfac620ac8c6eb7411233352691b09ad6e7 isn't correct.  `.venv` should be above `*.egg-info` and you shouldn't need the `/` that you currently have at the end of `.venv`.

---

@dav3r resolved with [071d1bb](https://github.com/cisagov/windows-commando-packer/pull/10/commits/071d1bb327dec20419467df4784c013534dd3366)